error-handler.js

```diff
@@ -269,6 +277,9 @@ createHandler = function createHandler(options) {
       return resumeOrClose(status);
     }
 
+    // Always set a response status.
```

Comment:
> "This will throw an error and crash the server if the response has already been sent"
While this condition is true, this doesn't apply to this code. The headers are never being set after the response is sent.

> "I'm pretty sure Express will default to a 200 response if you send() without a status"
This is also true. This is what is happening in the library right now. Unless you are lucky enough to get `renderDefault`, your response will be sent with a 200 (example: Try a static 404). All the response cases must be covered, and that's what that code does. My use case must send a 404 when serving a static 404 page, so that code must persist in some form.

In another Issue/PR we should go further and remove all the downstream status setting. I don't see the purpose. On the one case where you want to conform all the rest of the responses to 500, that can be done in addition. Like I previously stated, the response has not yet been sent.